barretenberg/cpp/scripts/test_chonk_standalone_vks_havent_changed.sh

```diff
@@ -13,7 +13,7 @@ cd ..
 # - Generate a hash for versioning: sha256sum bb-chonk-inputs.tar.gz
 # - Upload the compressed results: aws s3 cp bb-chonk-inputs.tar.gz s3://aztec-ci-artifacts/protocol/bb-chonk-inputs-[hash(0:8)].tar.gz
 # Note: In case of the "Test suite failed to run ... Unexpected token 'with' " error, need to run: docker pull aztecprotocol/build:3.0
-pinned_short_hash="79b094d8"
+pinned_short_hash="c72a0487"
```

Comment:
For the record: this change is due to the removal of BN254 lookup tables. Although we were not using the BN254 tables directly in any stdlib function, we might still be initialising the static tables, removing which causes the vk to change.